inferno/io/volumetric/volume.py

```diff
@@ -69,7 +69,14 @@ def pad_volume(self, padding=None):
         padding = self.padding if padding is None else padding
         if padding is None:
             return self.volume
-        else:
+        else:            
+			#sorry, too lazy to deal with nested lists in yaml
+            #this is for the case when only one int is passed for each axis
+            if len(self.padding)==(self.volume.ndim):
+                new_padding = ((self.padding[0],)*2,)
+                for i in range (1, len(self.padding)):
+                    new_padding+=((self.padding[i],)*2,)
```

Comment:
Supporting symmetric padding only specified by one integer is fine.
However, I think the check is not correct.
If we have the default numpy padding syntax, we would have something like`self.padding = [[10, 10], [10, 10], [10, 20]]', i.e. `len(self.padding) == self.volume.ndim == 3`.
So currently you are applying symmetric padding even when the full padding width is given.

I think the best option would be to replace the whole thing by this two lines:
```py
assert_(all(isinstance(pad, (int, tuple, list)) for pad in self.padding), "Expect int or iterable", TypeError)
self.padding = [[pad, pad] if isinstance(int, pad) else pad for pad in self.padding]
```
This way we support a mix of symmetric and potentially asymmetric  padding